simd.dd

```diff
@@ -60,6 +60,9 @@ import core.simd;
 	the compiler is targetting. If a particular CPU family has varying
 	support for vector types, an additional runtime check may be necessary.
 	The compiler does not emit runtime checks; those must be done by the programmer.
+    For instance, the Windows version of DMD only implements support for SIMD when
+    compiling with the 64bit flag. If a program does not contain runtime checks,
```

Comment:
> You're right that the "the 64bit flag" could be replaced with -m64. I'm new to github (I literally joined to make this change) and it's not entirely obvious to me how to fix it.

I'm not sure you can fix it from the website alone, you'll probably need a Git client. [GitHub for Windows](https://windows.github.com/) should be simple enough for new Git users.

> I realized I was making a mistake and thought the documentation would be improved if other people know that SIMD might require adding a compiler flag.

OK, but we shouldn't conflate runtime errors with compiler errors.